elements/lisk-transaction-pool/src/transaction_pool.ts

```diff
@@ -174,9 +174,25 @@ export class TransactionPool {
 	}
 
 	public removeTransaction(tx: Transaction): boolean {
-		// FIXME: this is log to supress ts build error
-		console.log(tx);
-		return false;
+		const foundTx = this._allTransactions[tx.id];
+		if (!foundTx) {
+			return false;
+		}
+
+		delete this._allTransactions[tx.id];
+		this._transactionList[
+			getAddressFromPublicKey(foundTx.senderPublicKey)
+		].remove(tx.nonce);
+
+		// Remove from feePriorityQueue
+		this._feePriorityQueue.clear();
+		for (const node of this._feePriorityQueue.getAllNodes()) {
```

Comment:
I think this is basically same as recreating the new fee priority queue from `this._allTransactions`?
then i think we dont need to add `getAllNodes` and simply recreate?